docs/source/topics/data_types.rst

```diff
@@ -4,8 +4,790 @@
+
+.. code-block:: ipython
+
+  In [8]: structure.get_ase()
+  Out[8]: Atoms(symbols='Li2', pbc=True, cell=[3.5, 3.5, 3.5], masses=...)
+
+  In [9]: structure.get_pymatgen()
+  Out[9]:
+  Structure Summary
+  Lattice
+      abc : 3.5 3.5 3.5
+  angles : 90.0 90.0 90.0
+  volume : 42.875
+        A : 3.5 0.0 0.0
+        B : 0.0 3.5 0.0
+        C : 0.0 0.0 3.5
+  PeriodicSite: Li (0.0000, 0.0000, 0.0000) [0.0000, 0.0000, 0.0000]
+  PeriodicSite: Li (1.5000, 1.5000, 1.5000) [0.4286, 0.4286, 0.4286]
+
+Exporting
+^^^^^^^^^
+
+The following export formats are available for :py:class:`~aiida.orm.nodes.data.structure.StructureData`:
+
+* ``xsf`` (format supported by e.g. XCrySDen and other visualization software;
+  supports periodic cells)
+* ``xyz`` (classical xyz format, does not typically support periodic cells (even if
+  the cell is indicated in the comment line)
+* ``cif`` (export to CIF format, without symmetry reduction, i.e. always storing the
+  structure as P1 symmetry)
+
+The node can be exported using the verdi CLI, for example:
+
+.. code-block:: console
+
+    $ verdi data structure export --format xsf <IDENTIFIER> > Li.xsf
+
+Where ``<IDENTIFIER>`` is one of the possible identifiers of the node, e.g. its PK or UUID.
+This outputs the structure in ``xsf`` format and writes it to a file.
+
+.. _topics:data_types:materials:trajectory:
+
+TrajectoryData
+--------------
+
+The :py:class:`~aiida.orm.nodes.data.array.trajectory.TrajectoryData` data type represents a sequences of StructureData objects, where the number of atomic kinds and sites does not change over time.
+Beside the coordinates, it can also optionally store velocities.
+If you have a list of :py:class:`~aiida.orm.nodes.data.structure.StructureData` instances called ``structure_list`` that represent the trajectory of your system, you can create a :py:class:`~aiida.orm.nodes.data.array.trajectory.TrajectoryData` instance from this list:
+
+.. code-block:: ipython
+
+  In [1]: TrajectoryData = DataFactory('array.trajectory')
+
+  In [2]: trajectory = TrajectoryData(structure_list)
+
+Exporting
+^^^^^^^^^
+
+You can export the py:class:`~aiida.orm.nodes.data.array.trajectory.TrajectoryData` node with ``verdi data trajectory export``, which accepts a number of formats including ``xsf`` and  ``cif``, and additional parameters like ``--step NUM`` (to choose to export only a given trajectory step).
+
+The following export formats are available:
+
+* ``xsf`` (format supported by e.g. XCrySDen and other visualization software;
+  supports periodic cells)
+* ``cif`` (export to CIF format, without symmetry reduction, i.e. always storing the
+  structures as P1 symmetry)
+
+.. todo::
+
+  Think of more example use cases.
+
```

Comment:
Do we want to keep a todo in the docs? Maybe better to open an issue?

Reply:
I've opened an issue on adding the missing data types to this section in #4529. This todo is just to make sure I also expand the usage section on `TrajectoryData` a bit when I work on these.

Reply:
I think what we were currently doing is both. So maybe add a reference to that PR in the `.. todo::` (see other sections like `internals/engine.rst` for reference) and also mention in the issue the specific change expected here.

Reply:
👍  Alright, thanks @ramirezfranciscof!